modules/Campaigns/Campaign.php

```diff
@@ -436,4 +437,4 @@ function getDeletedCampaignLogLeadsCount()
         return (int)$result['count'];
     }
 }
-?>
\ No newline at end of file
+?>
```

Comment:
PSR-2 requires that we do not close tags for class files

Reply:
I didn't know you guys were now following PSR-2. I would be happy to remove that for you.

Reply:
@daniel-samson Are you only removing them from "class" files? Or from all .php files?

Reply:
Hi @isleshocky77, 

We have been focusing on the PSR standards since around version 7.8.x. SuiteCRM is a rather large application, so it will take time to fix all of the code base to comply with PSR-2.

When a contributor makes a pull request, we have been asking each contributor to fix the lines in which they have touched. This is especially true, when you are adding new code to the project. This is because we do not wish to add any more issues to project. We currently use inspection or static analysis tools to work out our progress.

Also,
We have changed our automated testing processes so that we will catch errors much faster. We have introduced composer into 7.10. So there will be more focus on the PSR standards in the future.

Our coding guidelines, is basically just the PSR-2 guidelines with "how to add new license headers".

Reply:
Here is our progress so far here: https://sonarcloud.io/dashboard?id=SuiteCRM